### DISCUSSION TITLE
Add horizontal lines around "all" Jupyter cells

### DISCUSSION THREAD

**Discussion by ma-sadeghi** [microsoft/vscode-jupyter#11339] [New Ideas]
Currently, only the active cell with the cursor is surrounded with horizontal lines. It'd be nice for visibility to add them to "all" the cells, not just the active one. Jupyter does this and I think it makes it more organized. Here's a screenshot of what VS Code currently does:

![image](https://user-images.githubusercontent.com/14086031/189297146-bcd68041-ac2b-4631-b90e-2c6845d2aa3f.png)

Thanks!

**Comment by amunger** [comment]
from https://github.com/microsoft/vscode-jupyter/issues/6606, it looks like we used to have grey lines for inactive cells (and that was too noisy for some). We could look into trying to enable that again through a setting

**Reply by ma-sadeghi** [reply]
That'd be great if it could be switched on/off via a setting. For what it's worth, if someone doesn't like the cells to be visually separated, they might as well not use the cell syntax (%%). After all, one of the main reasons for using the cell syntax is so we can break up a long piece of code into individual chunks that can be individually run.

**Reply by amunger** [reply]
created https://github.com/microsoft/vscode-jupyter/issues/11469 to track this
